docs/v2/spec/2018_Modified_LAR_Code_Sheet.csv

```diff
@@ -0,0 +1,246 @@
+25,Race of Applicant or Borrower,42,Guamanian or Chamorro,
+25,Race of Applicant or Borrower,43,Samoan,
+25,Race of Applicant or Borrower,44,Other Pacific Islander,
+25,Race of Applicant or Borrower,5,White,
+25,Race of Applicant or Borrower,6,"Information not provided by applicant in mail, internet, or telephone application. ",Note Code 6 is only valid for Race of Applicant or Borrower: 1. 
+25,Race of Applicant or Borrower,7,Not applicable. ,Note Code 7 is only valid for Race of Applicant or Borrower: 1. 
+30,Race of Co-Applicant or Co-Borrower,1,American Indian or Alaska Native,
+30,Race of Co-Applicant or Co-Borrower,2,Asian,
+30,Race of Co-Applicant or Co-Borrower,21,Asian Indian,
+30,Race of Co-Applicant or Co-Borrower,22,Chinese,
+30,Race of Co-Applicant or Co-Borrower,23,Filipino,
+30,Race of Co-Applicant or Co-Borrower,24,Japanese,
+30,Race of Co-Applicant or Co-Borrower,25,Korean,
+30,Race of Co-Applicant or Co-Borrower,26,Vietnamese,
+30,Race of Co-Applicant or Co-Borrower,27,Other Asian,
+30,Race of Co-Applicant or Co-Borrower,3,Black or African American,
+30,Race of Co-Applicant or Co-Borrower,4,Native Hawaiian or Other Pacific Islander,
+30,Race of Co-Applicant or Co-Borrower,41,Native Hawaiian,
+30,Race of Co-Applicant or Co-Borrower,42,Guamanian or Chamorro,
+30,Race of Co-Applicant or Co-Borrower,43,Samoan,
+30,Race of Co-Applicant or Co-Borrower,44,Other Pacific Islander,
+30,Race of Co-Applicant or Co-Borrower,5,White,
+30,Race of Co-Applicant or Co-Borrower,6,"Information not provided by applicant in mail, internet, or telephone application. ",Note Code 6 is only valid for Race of Co-Applicant or Co-Borrower: 1
+30,Race of Co-Applicant or Co-Borrower,7,Not applicable. ,Note: Code 7 is only valid for Race of Co-Applicant or Co-Borrower: 1
+30,Race of Co-Applicant or Co-Borrower,8,No co-applicant. ,Note: Code 8 is only valid for Race of Co-Applicant or Co-Borrower: 1
+35,Race of Applicant or Borrower Collected on the Basis of Visual Observation or Surname,1,Collected on the basis of visual observation or surname,
+35,Race of Applicant or Borrower Collected on the Basis of Visual Observation or Surname,2,Not collected on the basis of visual observation or surname,
+35,Race of Applicant or Borrower Collected on the Basis of Visual Observation or Surname,3,Not applicable,
+36,Race of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname,1,Collected on the basis of visual observation or surname,
+36,Race of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname,2,Not collected on the basis of visual observation or surname,
+36,Race of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname,3,Not applicable ,
+36,Race of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname,4,No co-applicant. ,Note: Code 4 is only valid for Race of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname

```

Comment:
Is this note to differentiate it from the applicant version of the field? If so, we don't need a note here as this is a separate data field.

Reply:
Agreed, and will remove for the others too.